.github/workflows/requirements.txt

```diff
@@ -1,5 +1,5 @@
-lark-parser==0.8.1
-fastapi==0.52.0
+lark-parser==0.8.5
+fastapi==0.53.1
```

Comment:
I don't think Dependabot sees these files, only the `setup.py` file.
At least I didn't add these files, since I wasn't sure if that was wise. I can try though?

Reply:
It looks like it updated these files in #219, hence my confusion. I think we should be updating these versions to track the highest supported version in `setup.py`.

Reply:
I see your confusion. I believe those edits were mine, on top of the dependabot PR.

Reply:
Created issue #249 to track this.
I will set this up _after_ this PR.